### PR TITLE
Fixed the Pipfile generation syntax and added tests

### LINKED ISSUE #86: Pipfile: failed to render version a_package>=1.0.0
```
[[source]]
url = 'https://pypi.python.org/simple'
verify_ssl = true
name = 'pypi'

[requires]
python_version= '3.6'

[packages]
    pyecharts>=1.0.0 <-- error

[dev-packages]
nose = "*"
mock = "*"
codecov = "*"
coverage = "*"
flake8 = "*"
```

pipenv throws the following error:
`toml.decoder.TomlDecodeError: Found invalid character in key name: '>'. Try quoting the key name. (line 10 column 14 char 141)`

### PR DESCRIPTION
**Pull request by siddhpant**
Closes #86 

Should hopefully work now.

### CODE REVIEW

**Comment by siddhpant** [comment]
Detached HEAD problem in Travis CI

File: templates/Pipfile.jj2 [hunk 1]
```diff
@@ -3,18 +3,41 @@ url = 'https://pypi.python.org/simple'
...
 
+
 [packages]
 {% for dependency in dependencies: %}
-  {% if ('=' in dependency) or ('>' in dependency) or ('<' in dependency) %}
-    {{dependency}}
+
+  {% if ('=' in dependency) %}
+    {% with dep, ver = dependency.split('=') %}
+      {{dep}} = '={{ver}}
```
Comment (chfw) [inline templates/Pipfile.jj2]: should each line align to the left?

**Comment by chfw** [comment]
please update your branch from dev and travis shall pass the test

File: templates/Pipfile.jj2 (hunk 1 above)
Comment (siddhpant) [reply templates/Pipfile.jj2]: Fixed.

**Comment by chfw** [comment]
1. would you please contribute a test case? like what we have here: https://github.com/moremoban/pypi-mobans/tree/dev/tests

2. put a line in changelog

**Comment by siddhpant** [comment]
Okay

**Comment by siddhpant** [comment]
Would fix error tomorrow, it's 22:21 here 😅

**Comment by siddhpant** [comment]
Done. Tests passed.

**Pull request merged**